src/containers/TitleValue.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import TitleValueComponent from '../components/TitleValue';
+
+class TitleValue extends Component {
+  render() {
+    console.log(this.props.label.label);
+    return <TitleValueComponent label={ this.props.label.label} initialValue={this.props.number}/>;
+  }
```

Comment:
Here i just need access to props as i am not dispatching anything. How can i achieve this ?

Reply:
Here is a caveat; if there is no state or dispatch then there should only be a component. Containers are reserved for state and dispatch management.

Reply:
If i do not have the connect present, how can the props be passed.?

Reply:
From the recording, In order to have access to state we need to use the connect function. Since, we do not dispatch anything here we can remove  mapDispatchToProps.

Reply:
Yes you are correct.  To only get the props you would use connect with mapStateToProps, and no mapDispatchToProps